src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -2048,16 +2048,28 @@ export default class Aircraft {
                 const dist_to_localizer = offset[0] / sin(angle_diff); // dist from the localizer intercept point, km
 
                 if (dist_to_localizer <= turning_radius || dist_to_localizer < 0.5) {
-                    this.target.heading = angle;
-
                     // Steer to within 3m of the centerline while at least 200m out
                     if (offset[1] > 0.2 && abs(offset[0]) > 0.003) {
-                        // TODO: enumerate the magic numbers
-                        this.target.heading = _clamp(degreesToRadians(-30), -12 * offset_angle, degreesToRadians(30)) + angle;
+                        const severity_of_correction = 12;  // controls steepness of heading adjustments during localizer tracking
+                        const tgtHdg = angle + (offset_angle * -severity_of_correction);
+                        let minHdg = null;
+                        let maxHdg = null;
+                        if (!assignedHdg) {  // interception via fixes
+                            // TODO: uh.... do this.
```

Comment:
One thing my mentors always ask ask is who is the audience for this error?
- Does the user need to know about it?
- Or is this meant for developers?

Once you figure out the audience, then you need to decide on how to present it to that audience.  

In this instance, I would probably go with a `console.warn()` with a short message.  Will an error here actually break anything, like cause other parts of the app to fail?  Or will the app keep plugging away?

I'd also fill out that `TODO` block with a little more detail about what needs to be done.  Doesn't need to be a book, just a quick note so the next person that deals with it knows why there is an empty if block and what the future may hold for said block.